packages/greenbox-script/src/generateIotms.ts

```diff
@@ -0,0 +1,17 @@
+import { Item, print, toInt } from "kolmafia";
```

Comment:
This probably wants to live elsewhere - we could make a greenbox-tools package for this stuff maybe

Reply:
With its own build.mjs

Reply:
oops… yeah sorry i meant to delete this before the push. my b